src/indicators/flooding/glofas/glofas.py

```diff
@@ -303,11 +311,15 @@ def download(
         area: Area,
         leadtimes: List[int],
         version: int = DEFAULT_VERSION,
+        year_min: int = None,
+        year_max: int = None,
     ):
```

Comment:
I would say it is nice to have the year option, as I can see use cases where you are only interested in a couple of years and since the downloading is taking so long

Reply:
Okay I will leave it then, thanks for the feedback :)